README.md

```diff
@@ -1,120 +1,110 @@
+to an event within Slack.
 
-A [link trigger](https://api.slack.com/future/triggers/link) is a type of
+When you `run` or `deploy` your project for the first time, the CLI will prompt
+you to create a trigger if one is found in the `triggers/` directory. For any
+subsequent triggers added to the application, each must be
+[manually added using the `trigger create` command](#manual-trigger-creation).
+
+When creating triggers, you must select the workspace and environment that you'd
+like to create the trigger in. Each workspace can have a local development
+version (denoted by `(local)`), as well as a deployed version. _Triggers created
+in a local environment will only be available to use when running the
+application locally._
+
+### Link Triggers
+
+A [link trigger](https://api.slack.com/automation/triggers/link) is a type of
 trigger that generates a **Shortcut URL** which, when posted in a channel or
 added as a bookmark, becomes a link. When clicked, the link trigger will run the
 associated workflow.
 
 Link triggers are _unique to each installed version of your app_. This means
 that Shortcut URLs will be different across each workspace, as well as between
 [locally run](#running-your-project-locally) and
-[deployed apps](#deploying-your-app). When creating a trigger, you must select
-the Workspace that you'd like to create the trigger in. Each Workspace has a
-development version (denoted by `(dev)`), as well as a deployed version.
+[deployed apps](#deploying-your-app).
 
-To create a Link Trigger for the **Log a run** workflow in this sample, run the
-following command:
+With link triggers, after selecting a workspace and environment, the output
+provided will include a Shortcut URL. Copy and paste this URL into a channel as
+a message, or add it as a bookmark in a channel of the workspace you selected.
+Interacting with this link will run the associated workflow.
 
-```zsh
-$ slack trigger create --trigger-def triggers/log_run_trigger.ts
-```
+**Note: triggers won't run the workflow unless the app is either running locally
+or deployed!**
 
-After selecting a Workspace, the output provided will include the link trigger
-shortcut URL. Copy and paste this URL into a channel as a message, or add it as
-a bookmark in a channel of the Workspace you selected.
+### Manual Trigger Creation
 
-**Note: this link won't run the workflow until the app is either running locally
-or deployed!** Read on to learn how to run your app locally and eventually
-deploy it to Slack hosting.
-
-## Running Your Project Locally
-
-While building your app, you can see your changes propagated to your workspace
-in real-time with `slack run`. In both the CLI and in Slack, you'll know an app
-is the development version if the name has the string `(dev)` appended.
+To manually create a trigger, use the following command:
 
 ```zsh
-# Run app locally
-$ slack run
-
-Connected, awaiting events
+$ slack trigger create --trigger-def triggers/log_run_trigger.ts
```

Comment:
It may be nice to include both triggers to use this app (log a run and display the leaderboard of runs):
```suggestion
$ slack trigger create --trigger-def triggers/log_run_trigger.ts
$ slack trigger create --trigger-def triggers/display_leaderboard_trigger.ts
```